fabtests/benchmarks/dgram_pingpong.c

```diff
@@ -55,6 +56,8 @@ static int run(void)
 	ret = fi_recv(ep, rx_buf, rx_size + ft_rx_prefix_size(), mr_desc,
 			0, &rx_ctx);
 
+	if (ret)
+		return ret;
```

Comment:
nit: Can you move the blank line above the if to below the return?

Reply:
Done in the new revision